GoogleForCreators/web-stories-wp
# Autosaves: delete local auto-draft autosave after save

## Context



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create new story
2. Wait for local autosave -> `wp_stories_autosave_story_auto-draft` gets created
3. Make proper save
4. Make more changes
5. Wait for local autosave -> `wp_stories_autosave_story_12345` gets created
6. Verify that `wp_stories_autosave_story_auto-draft` is discarded at this point


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12342
